include/core/mu_parser_internal.h

```diff
@@ -0,0 +1,206 @@
+/* ---------------------------------------------------------------------
+ *
+ * Copyright (C) 2019 -  by the Lethe authors
+ *
+ * This file is part of the Lethe library
+ *
+ * The Lethe library is free software; you can use it, redistribute
+ * it, and/or modify it under the terms of the GNU Lesser General
+ * Public License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ * The full text of the license can be found in the file LICENSE at
+ * the top level of the Lethe distribution.
+ *
+ * ---------------------------------------------------------------------
+ */
+
+#ifndef lethe_mu_parser_internal_h
+#define lethe_mu_parser_internal_h
+
+#include <deal.II/base/config.h>
+
+#include <deal.II/base/array_view.h>
+#include <deal.II/base/mu_parser_internal.h>
+#include <deal.II/base/tensor.h>
+#include <deal.II/base/thread_local_storage.h>
+#include <deal.II/base/thread_management.h>
+#include <deal.II/base/utilities.h>
+
+#include <muParser.h>
+#include <muParserFixes.h>
+
+#include <cmath>
+#include <ctime>
+#include <limits>
+#include <map>
+#include <memory>
+#include <mutex>
+#include <random>
+#include <string>
+#include <vector>
+
+namespace internal
+{
+  namespace FunctionParserCustom
+  {
+    int
+    mu_round(double val);
+
+    double
```

Comment:
Not sure, but should the namespace also have the /// field description?

Reply:
I added it also, with the doc for each function